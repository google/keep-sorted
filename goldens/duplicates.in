Remove duplicates:
  // keep-sorted-test start remove_duplicates=yes
  foo
  bar
  foo
  // keep-sorted-test end

XXX: This behavior is incorrect.
Remove duplicates, case insensitive:
  // keep-sorted-test start remove_duplicates=yes case=no
  foo
  FoO
  bar
  // keep-sorted-test end

Remove duplicates with comments:
  // keep-sorted-test start remove_duplicates=yes sticky_comments=yes
  foo
  bar
  // Foo again
  foo
  // Foo again
  foo
  // keep-sorted-test end

Preserve duplicates:
  // keep-sorted-test start remove_duplicates=no
  foo
  bar
  foo
  // keep-sorted-test end
