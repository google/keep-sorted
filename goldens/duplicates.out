Remove duplicates:
  // keep-sorted-test start remove_duplicates=yes
  bar
  foo
  // keep-sorted-test end

XXX: This behavior is incorrect.
Remove duplicates, case insensitive:
  // keep-sorted-test start remove_duplicates=yes case=no
  bar
  FoO
  foo
  // keep-sorted-test end

Remove duplicates with comments:
  // keep-sorted-test start remove_duplicates=yes sticky_comments=yes
  bar
  foo
  // Foo again
  foo
  // keep-sorted-test end

Preserve duplicates:
  // keep-sorted-test start remove_duplicates=no
  bar
  foo
  foo
  // keep-sorted-test end
